ivy/functional/frontends/numpy/mathematical_functions/arithmetic_operations.py

```diff
@@ -1,71 +0,0 @@
-# global
-import ivy
```

Comment:
Please revert all changes made to this file as it's not related to the function you're working on